### PR TITLE
Use `#[\SensitiveParameter]`

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: src/OpenAI/Platform/Azure.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ public function __construct(
         private string $baseUrl,
         private string $deployment,
         private string $apiVersion,
-        private string $key,
+        #[\SensitiveParameter] private string $apiKey,
```
Comment (OskarStark) [inline src/OpenAI/Platform/Azure.php:21]: Needs to be adjusted in bundle config

Comment (chr-hertel) [reply src/OpenAI/Platform/Azure.php:21]: after the next tag the bundle needs some love anyways :+1:

Comment (OskarStark) [reply src/OpenAI/Platform/Azure.php:21]: Prepared in
* https://github.com/php-llm/llm-chain-bundle/pull/38

**Review by chr-hertel — APPROVED** [review]
good catch :+1:

File: src/OpenAI/Platform/Azure.php (hunk 1 above)
Comment (OskarStark) [reply src/OpenAI/Platform/Azure.php:21]: > after the next tag the bundle needs some love anyways 👍

Yes, lets do it in
* https://github.com/php-llm/llm-chain-bundle/pull/38

**Pull request merged**